tables/en-ueb-g1.ctb

```diff
@@ -35,16 +35,16 @@ begcaps 6-6
 endcaps 6-3
 
 # Typeforms (rulebook 9)
-firstwordital 46-2345 # Start of italicized passage.
-lastworditalbefore 46-3
+firstwordital 46-2356 # Start of italicized passage.
+lastworditalafter46-3
```

Comment:
Sorry corrected!

On 20/05/2015 10:47, Bert Frees wrote:
>
> In tables/en-ueb-g1.ctb 
> <https://github.com/liblouis/liblouis/pull/79#discussion_r30685523>:
>
> > @@ -35,16 +35,16 @@ begcaps 6-6
> >  endcaps 6-3
> >
> >  # Typeforms (rulebook 9)
> > -firstwordital 46-2345 # Start of italicized passage.
> > -lastworditalbefore 46-3
> > +firstwordital 46-2356 # Start of italicized passage.
> > +lastworditalafter46-3
>
> looks like a little typo here
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/liblouis/liblouis/pull/79/files#r30685523>.
>

-- 
Paulw.torchtrust signature

Paul Wood, Chief Technical Officer
*Torch Trust*
Torch House, Torch Way,
Market Harborough, Leics. LE16 9HL, UK
Direct Line: *+44(0)1858 438269*
Tel: *+44(0)1858 438260*, Fax: *+44(0)1858 438275*
Email: paulw@torchtrust.org <mailto:paulw@torchtrust.org>
Website: www.torchtrust.org <http://www.torchtrust.org/>

____________________________________________________

Chief Executive: Dr Gordon Temple
Charity No. 1095904

Privileged/Confidential Information may be contained in this message.
If you are not the intended recipient please destroy this message
and kindly notify the sender by reply email. The computer from which
this mail originates is equipped with virus screening software.
However Torch Trust cannot guarantee that the mail and its attachments
are free from virus infection.